pkg/client/core/store/filesystem/identitymanager.go

```diff
@@ -57,6 +57,29 @@ type IdentityPoolUser struct {
 	StackName  string
 }
 
+// RemoveIdentityPool removes alias, cert, and indentitypool files from file store
```

Comment:
indentitypool -> identitypool

unless there is some cool new aws service that indents for you

/funny

Reply:
Bah! :D